packages/@jsii/python-runtime/tests/test_compliance.py

```diff
@@ -1168,3 +1168,9 @@ def test_collection_of_interfaces_map_of_structs():
 def test_collection_of_interfaces_map_of_interfaces():
     for elt in InterfaceCollections.map_of_interfaces().values():
         assert getattr(elt, "ring") is not None
+
+
+@pytest.mark.skip # Currently broken because submodule names aren't case-adjusted correctly :(
```

Comment:
This is a pre-existing bug I'm merely shedding some light on. In order to fix this, I believe we'll need the kind of work in #1502 - specifically modeling the *submodules* information (including of dependencies) as part of the Assembly, so we can tell a PascalCased submodule name apart from a nested type's parent.